website/docs/reference/resource-configs/delimiter.md

```diff
@@ -0,0 +1,76 @@
+---
+resource_types: [seeds]
+datatype: <string>
+default_value: ","
+---
+
+## Definition
+
+An optional seed configuration used to separate values in a [seed](/docs/build/seeds) with a string you provide. The delimiter defaults to comma.
+
+
+## Usage
+Specify delimiter in your `dbt_project.yml` file to globally separate all seed values:
+
+<File name='dbt_project.yml'>
+
+```yml
+seeds:
+  +quote_columns: False
+  +delimiter: ","
```

Comment:
```suggestion
  +delimiter: ";"
```
Comma is the default, so I'd probably show an example of overriding the default here